include/launch_kernel.cuh

```diff
@@ -309,37 +126,11 @@ default:								\
   case 144: kernel<144, __VA_ARGS__><<<tp.grid, tp.block, tp.shared_bytes, stream>>>(arg); break;                      \
   case 192: kernel<192, __VA_ARGS__><<<tp.grid, tp.block, tp.shared_bytes, stream>>>(arg); break;                      \
   case 200: kernel<200, __VA_ARGS__><<<tp.grid, tp.block, tp.shared_bytes, stream>>>(arg); break;                      \
+  case 250: kernel<250, __VA_ARGS__><<<tp.grid, tp.block, tp.shared_bytes, stream>>>(arg); break;                      \
```

Comment:
Since we now know `5`s with `2`s are in play, we should also add `125` (`5 5 5 2`) and `50` (`5 5 2 2`). (`100` is already covered.) I guess I could make an argument for `40` and `80` as well.

Reply:
That is completely orthogonal to this PR and can be covered elsewhere .